tests/nlu/base/test_evaluation.py

```diff
@@ -275,12 +276,25 @@ def test_run_evaluation(unpacked_trained_moodbot_path):
         data, os.path.join(unpacked_trained_moodbot_path, "nlu"), errors=False
     )
     assert result.get("intent_evaluation")
-    assert result.get("entity_evaluation").get("DIETClassifier")
```

Comment:
should we keep tests on DIET? When we remove deprecated algorithms, we still need these tests, no?

Reply:
will add those again in another PR